### PR TITLE
fixes leftout eslint issues

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
# Description
fixes eslint issues for left-out files
 

## Related Issue
fixes#151

## How Has This Been Tested
1.  Integration test
2.  By running UI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by LijuJoseJJ** [comment]
this PR depends on PR #144

File: docker-entrypoint-initdb.d/01_setup_admin_user.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
-/* eslint-disable no-undef */
+/* eslint-disable no-undef, eqeqeq */
```
Comment (ocgordon) [inline docker-entrypoint-initdb.d/01_setup_admin_user.js]: what is this?

File: integration-test/test.js [hunk 2]
```diff
@@ -1,43 +1,36 @@
+/* eslint-disable import/no-extraneous-dependencies, no-undef */
```
Comment (ocgordon) [inline integration-test/test.js]: Can this be refactored so eslint doesn't have to be disabled?

File: integration-test/test.js [hunk 3]
```diff
@@ -1,43 +1,36 @@
...
+    await testData.configureDependentTestData();
+    ({ erc721, erc721Commitment, erc20Commitments } = testData);
   });
   /*
-  *  Step 1.
-  *  This Suite will create Alice and Bob.
-  */
-  describe('*** Create Users ***', async function () {
+   *  Step 1.
+   *  This Suite will create Alice and Bob.
```
Comment (ocgordon) [inline integration-test/test.js]: This step will create accounts for Alice and Bob.

File: integration-test/test.js [hunk 4]
```diff
@@ -61,19 +54,19 @@ describe('****** Integration Test ******\n', function () {
...
         });
     });
   });
   /*
-  * Step 2.
-  * This Suite will login Alice and Bob/
-  */
-  describe('*** Login Users ***', function () {
+   * Step 2.
+   * This Suite will login Alice and Bob/
```
Comment (ocgordon) [inline integration-test/test.js]: This step will log in Alice and Bob.

File: integration-test/test.js [hunk 5]
```diff
@@ -103,27 +96,27 @@ describe('****** Integration Test ******\n', function () {
...
-  * Story line:
-  *  Alice minted a ERC-721 token. She then Shield(mint ERC-721 commitment) that token
-  *  and transfer to Bob. He then burns received ERC-721 commitment,
-  *  and then transferred resultant ERC-721 token to Alice.
-  * finally, Alice burns received ERC-721 token
-  */
-  describe('*** ERC-721 and ERC-721 Commitment ***', function () {
-    context('Alice\'s tasks: ', function () {
+   * Step 3 to 8.
+   * This Suite will test ERC-721 & ERC-721 commitment
```
Comment (ocgordon) [inline integration-test/test.js]: This step will test the creation of ERC-721 tokens and ERC-721 token commitments.

File: integration-test/test.js [hunk 6]
```diff
@@ -103,27 +96,27 @@ describe('****** Integration Test ******\n', function () {
...
+   * Mint, Transfer and Burn features.
+   * Story line:
+   *  Alice minted a ERC-721 token. She then Shield(mint ERC-721 commitment) that token
```
Comment (ocgordon) [inline integration-test/test.js]: Alice mints an ERC-721 token. She then shields that token by minting an ERC-721 commitment and transfers that commitment to Bob.

File: integration-test/test.js [hunk 7]
```diff
@@ -103,27 +96,27 @@ describe('****** Integration Test ******\n', function () {
...
+   *  and transfer to Bob. He then burns received ERC-721 commitment,
```
Comment (ocgordon) [inline integration-test/test.js]: Bob then burns the received ERC-721 commitment and transfers the resulting ERC-721 token to Alice. Finally, Alice burns the received ERC-721 token.

File: integration-test/test.js [hunk 8]
```diff
@@ -191,21 +184,21 @@ describe('****** Integration Test ******\n', function () {
...
     });
-    context('Bob\'s tasks: ', function () {
+    context(`${bob.name} tasks: `, function() {
       /*
-      * This act as a delay.
-      * Which is needed, presuming till then transfree Whisper will able receive transferred data.
-      */
-      before((done) => setTimeout(done, 10000));
+       * This act as a delay.
+       * Which is needed, presuming till then transfree Whisper will able receive transferred data.
```
Comment (ocgordon) [inline integration-test/test.js]: This acts as a delay, which is needed to ensure that the recipient will be able to receive transferred data through Whisper.

File: integration-test/test.js [hunk 9]
```diff
@@ -246,21 +239,21 @@ describe('****** Integration Test ******\n', function () {
...
     });
-    context('Alice\'s tasks: ', function () {
+    context(`${alice.name} tasks: `, function() {
       /*
-      * This act as a delay.
-      * Which is needed, presuming till then transfree Whisper will able receive transferred data.
-      */
-      before((done) => setTimeout(done, 10000));
+       * This act as a delay.
+       * Which is needed, presuming till then transfree Whisper will able receive transferred data.
```
Comment (ocgordon) [inline integration-test/test.js]: This acts as a delay, which is needed to ensure that the recipient will be able to receive transferred data through Whisper.

File: integration-test/test.js [hunk 10]
```diff
@@ -274,48 +267,48 @@ describe('****** Integration Test ******\n', function () {
...
-  describe('*** ERC-20 and ERC-20 Commitment ***', function () {
-    context('Alice\'s tasks: ', function () {
+   * Step 9 to 16.
+   * This Suite will test ERC-20 & ERC-20 commitment
+   * Mint, Transfer and Burn features.
+   * Story line:
+   *  Alice minted a 5 ERC-20 token. She then Shield(mint ERC-20 commitment) two token with value 2 and 3
+   *  and transfer 4 token to Bob. He then burns received ERC-20 commitment,
+   *  and then transferred resultant  4 ERC-20 token to Alice.
+   * finally, Alice burns her received and change, 5 ERC-20 token
```
Comment (ocgordon) [inline integration-test/test.js]: Steps 9 through 16. 
This step will test the creation of ERC-20 tokens and ERC-20 token commitments, as well as the transfer and burning of these tokens and their commitments.
Alice mints 5 ERC-20 tokens. She then shields these tokens by creating 2 ERC-20 commitments with values of 2 and 3 tokens. Alice then transfers 4 ERC-20 tokens in commitments to Bob. Bob burns the received ERC-20 commitment and transfers the resulting 4 ERC-20 tokens to Alice. Finally, Alice burns her received ERC-20 tokens and her remaining ERC-20 token commitment.

File: integration-test/test.js [hunk 11]
```diff
@@ -423,21 +423,21 @@ describe('****** Integration Test ******\n', function () {
...
     });
-    context('Bob\'s tasks: ', function () {
+    context(`${bob.name} tasks: `, function() {
       /*
-      * This act as a delay.
-      * Which is needed, presuming till then transfree Whisper will able receive transferred data.
-      */
-      before((done) => setTimeout(done, 10000));
+       * This act as a delay.
+       * Which is needed, presuming till then transfree Whisper will able receive transferred data.
```
Comment (ocgordon) [inline integration-test/test.js]: This acts as a delay, which is needed to ensure that the recipient will be able to receive transferred data through Whisper.

File: integration-test/test.js [hunk 12]
```diff
@@ -477,21 +477,21 @@ describe('****** Integration Test ******\n', function () {
...
     });
-    context('Alice\'s tasks: ', function () {
+    context(`${alice.name} tasks: `, function() {
       /*
-      * This act as a delay.
-      * Which is needed, presuming till then transfree Whisper will able receive transferred data.
-      */
-      before((done) => setTimeout(done, 10000));
+       * This act as a delay.
+       * Which is needed, presuming till then transfree Whisper will able receive transferred data.
```
Comment (ocgordon) [inline integration-test/test.js]: This acts as a delay, which is needed to ensure that the recipient will be able to receive transferred data through Whisper.

File: integration-test/testData.js [hunk 13]
```diff
@@ -1,132 +1,137 @@
-import Utils from '../zkp-utils'
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (ocgordon) [inline integration-test/testData.js]: Can we rework so that eslint disable isn't necessary?

File: integration-test/testData.js [hunk 14]
```diff
@@ -1,132 +1,137 @@
...
 
-/*
-*  a function which will configure dependent test data.
-*/
-export const configureDependentTestData = (async function  () {
-  this.erc721Commitment = await this.erc721Commitment();
-  this.erc20Commitments = await this.erc20Commitments();
-}).bind(testData)
+  /*
+   *  a function which will configure dependent test data.
```
Comment (ocgordon) [inline integration-test/testData.js]: This function will configure dependent test data.

File: integration-test/test.js [hunk 15]
```diff
@@ -46,13 +39,13 @@ describe('****** Integration Test ******\n', function () {
...
           expect(res).to.have.nested.property('body.data.name');
-          done();
+          return done();
         });
     });
     /*
-    * Create account for bob.
-    */
-    it('Sign up Bob', function (done) {
+     * Create account for bob.
```
Comment (ocgordon) [inline integration-test/test.js]: Create an account for Bob.

File: integration-test/test.js [hunk 16]
```diff
@@ -1,43 +1,36 @@
...
-  describe('*** Create Users ***', async function () {
+   *  Step 1.
+   *  This Suite will create Alice and Bob.
+   */
+  describe('*** Create Users ***', async function() {
     /*
-    * Create account for Alice.
-    */
-    it('Sign up Alice', function (done) {
+     * Create account for Alice.
```
Comment (ocgordon) [inline integration-test/test.js]: Create an account for Alice.

File: integration-test/testData.js (hunk 13 above)
Comment (LijuJoseJJ) [reply integration-test/testData.js]: this has added to avoid error
```error  'config' should be listed in the project's dependencies, not devDependencies  import/no-extraneous-dependencies```
config module are used in some microservice and we are planing to use it in other microservice also.
but, for integration test - logic code reside in ``` integration-test/``` folder
which not run as microservice,
since is integration test is part of development phase, all it dependencies are installed as devDependencies
which explain config module installed devDependencies in nightfall/package.json

File: docker-entrypoint-initdb.d/01_setup_admin_user.js (hunk 1 above)
Comment (LijuJoseJJ) [reply docker-entrypoint-initdb.d/01_setup_admin_user.js]: this particular file is actually a js script which is running in a mongo shell.
two check of eslint is disabled explicitly for respective reason.
1. no-undef :  in the file variable db is used without declaration because db is globally assigned variable in mongo shell. If we try to assign it - script will fail.
2. eqeqeq: This check look for usage of double equal ('==')  in place it should be triple equal ('===')  as double equal do type conversion. But, in mongo shell triple equal is not doing job as expected
especially for if case
``` if (db == 'nightfall_test') ```

File: integration-test/test.js (hunk 2 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: well, this has added to avoid error such as
error 'config' should be listed in the project's dependencies, not devDependencies import/no-extraneous-dependencies
for integration test - logic code reside in integration-test/ folder
which not run as microservice,
since integration test is part of development phase, all it dependencies are installed as devDependencies (for example:- chai, superagent config .. etc)
which explain these module installed as devDependencies in nightfall/package.json.

so in short we need this check of eslint to be disabled

File: integration-test/test.js (hunk 3 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 4 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 6 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 5 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: change to
```
* Step 3 to 8.
* These step will test ERC-721 & ERC-721 commitment operations such as mint, transfer and burn.
```

File: integration-test/test.js (hunk 7 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 8 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 9 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 11 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 12 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/testData.js (hunk 14 above)
Comment (LijuJoseJJ) [reply integration-test/testData.js]: changes done

File: integration-test/test.js (hunk 15 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: integration-test/test.js (hunk 10 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: changes done

File: docker-entrypoint-initdb.d/01_setup_admin_user.js (hunk 1 above)
Comment (LijuJoseJJ) [reply docker-entrypoint-initdb.d/01_setup_admin_user.js]: case handled and eslint-disable removed

File: integration-test/test.js (hunk 2 above)
Comment (LijuJoseJJ) [reply integration-test/test.js]: case handled and eslint disable removed

File: integration-test/testData.js (hunk 13 above)
Comment (LijuJoseJJ) [reply integration-test/testData.js]: case handled and eslint disable removed

**Comment by ChaitanyaKonda** [comment]
code reviewed. tests passed
